### ISSUE TITLE
[Blog] - [Blog Page] - Related Topics

### ISSUE THREAD

**Issue by danielfmiranda** [mozilla/foundation.mozilla.org#8962]

**Comment by cdanfon** [comment]
@danielfmiranda to confirm with Nancy the expected behaviour

**Comment by danielfmiranda** [comment]
Hi @nancyt1!

@tbrlpld and I had a quick question about this ticket.

Just to double check, currently each blog post has a topics field that the author can link to it through the CMS, and these will appear next to the blog page title on both the actual blog page and the blog card.


Is this new requested "related topics" section just repeating the topics from above ^^? Or, is this a new field entirely, where users are picking other topics to display in this section?

Thanks in advance!

**Comment by cdanfon** [comment]
Hey @nancyt1 please see the above comment from @danielfmiranda.

**Comment by nancyt1** [comment]
Hey @danielfmiranda, This is a section where users can pick other topics to display!

**Comment by tbrlpld** [comment]
@nancyt1 Is this still necessary when a blog post can have more than one topic?

**Comment by tbrlpld** [comment]
So these are topics that the blog post does not belong to directly, but that are related?

**Comment by nancyt1** [comment]
@tbrlpld yes! since the max amount of topics a post can have is 2, it's just another opportunity to include more or at least work similar to how the "related posts" work now.

**Comment by tbrlpld** [comment]
@nancyt1 We just bumped the max from 1 to 2. There is no technical reason for this. We could make that more if that is desirable. We would need to reconsider how we are displaying the categories though.

**Comment by nancyt1** [comment]
@tbrlpld I'd like to keep it at 2 for now since I'm not aware of any use cases to need more than that but it would be good to revisit after seeing real usage.

**Comment by simont-cr** [comment]
Hi @nancyt1 and @kristinashu , as this is the last remaining ticket from the Blog Page epic #8786, wanted to double check if we're still planning on doing this or it is safe to close for now.
